addons/l10n_it_stock_ddt/report/l10n_it_ddt_report.xml

```diff
@@ -31,7 +31,7 @@
                         <div class="mt16"/>
                         <div class="mt64"/>
                         <div>
-                            <h1>Documento di Trasporto <span t-esc="o.l10n_it_ddt_number"/></h1>
+                            <h1>Transport Document <span t-esc="o.l10n_it_ddt_number"/></h1>
```

Comment:
I think we did this on purpose.  (because it is an official document called that way)